### ISSUE TITLE
questions about ephys data processing

### ISSUE THREAD

**Issue by hongruhu** [AllenInstitute/coupledAE-patchseq#3]
Hi,
In the paper (https://www.nature.com/articles/s43588-021-00030-1) you mentioned that the inhibitory neuron ephys data from Scala et al. is also used as a "test dataset" with the pre-trained model. I was wondering if there's any ways/ scripts to compute the same ipfx features and feature vectors of these datasets? Because from their mini-atlas GitHub repo, the features does not really match what the ipfx and feature vectors give.

Thanks!

Original paragraph from the coupledAE paper:
```We directly tested the idea that pre-trained coupled autoencoders can be used to predict unobserved cross-modal features in independent experiments by using two recent Patch-seq datasets22,23, which include 107 and 524 inhibitory neurons from the mouse motor cortex, respectively. We applied a coupled autoencoder trained on the dataset in this work without extra training to predict the transcriptomic labels and electrophysiological properties of these neurons from their transcriptomic profiles. The results in Supplementary Figs. 2 and 3, and Extended Data Figs. 8 and 9 show that this approach yields accurate prediction of cell-type labels and certain electrophysiological properties, despite a 5% mismatch between the gene lists, differences in electrophysiology protocols and brain regions.```

**Comment by rhngla** [comment]
The transcriptomic protocols across the datasets are similar enough to simply use Scala et al. gene expression values as input to our models that are trained only on the Gouwens et al. dataset. We therefore use the Scala et al. transcriptomic data to assess representations (through the ability of a classifier to predict T-type labels based on the representations), and to check if we can explain any of the variation in their ephys measurements when we use our model in the cross-modal (T-->E) setting.

Experimental conditions (e.g. temperature) and protocols (e.g. stimulus set) to measure ephys. responses in Scala et al. are quite different compared Gouwens et al. Therefore, some features cannot be calculated, and the relationship between other features may not be one-to-one. I have now [included a file](https://github.com/AllenInstitute/coupledAE-patchseq/blob/cplAE-TE/data/proc/E_feature_correspondences.txt) that indicates features that might be related across the two datasets (based on a conversation with Nathan Gouwens). 

Since the cross-modal setting only allows us to predict the Gouwens et al. ephys feature set (since that is what the model is training on), in Extended Figure 9 we show a subset of these potentially related features that have a high correlation with the measurements in Scala et al.

**Comment by hongruhu** [comment]
Thank you!
